app/controllers/api/v1/registrations_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class RegistrationsController < ApplicationController
+      skip_before_action :doorkeeper_authorize!
+
+      def create
+        validation = RegistrationForm.new(sign_up_params)
+        if validation.save
+          render success: true, status: :created
+        else
+          render json: { error: validation.errors.full_messages.to_sentence }, status: :unprocessable_entity
+        end
+      end
+
+      private
+
+      def sign_up_params
```

Comment:
```suggestion
      def create_params
```

params are usually related to a specific action, so reusing the name of this action makes it easier to understand the context ;-)